shared/structures/src/endpoints/UitpasPriceCheckRequest.ts

```diff
@@ -0,0 +1,17 @@
+import { AutoEncoder, StringDecoder, field, IntegerDecoder } from '@simonbackx/simple-encoding';
+
+export class UitpasPriceCheckRequest extends AutoEncoder {
+    @field({ decoder: IntegerDecoder })
+    price: number;
+
+    @field({ decoder: StringDecoder, nullable: true })
+    uitpasEventId: string;
```

Comment:
```suggestion
    @field({ decoder: StringDecoder, nullable: true })
    uitpasEventId: string | null = null;
```
Als nullable true is, moet de TypeScript type daar ook mee overeenkomen